client/src/components/Board.tsx

```diff
@@ -80,9 +83,43 @@ const Board = ({
     setGameState(getGameState(newBoard))
 
     setBoard(newBoard)
-    setTurn(false)
+
+    if (isRobotMode) {
+      if (!setPlayerValue) throw Error("Player value not defined")
+
+      if (turn === true) {
+        // It's human turn
+        setPlayerValue(PlayerValue.Two)
+      } else {
+        setPlayerValue(PlayerValue.One)
+      }
+      setTurn(!turn)
+    } else {
+      setTurn(false)
```

Comment:
Line 96 (player vs robot) toggles turn
Line 98 (player vs player) sets turn to false (it gets set to true only for other player [here](https://github.com/iamlordaubrey/connect4-game/blob/main/client/src/components/Game.tsx#L133))